src/index.html

```diff
@@ -9,10 +9,51 @@
     <title>Stars</title>
     <link
```

Comment:
Attributes of the `link` tag should start each on a new line with 2-space indentation related to the tag to keep the attributes correctly formatted.